### ISSUE TITLE
Uncaught Error permission_denied at Client doesn't have permission to access the desired data 

### ISSUE THREAD

**Issue by wheredoesyourmindgo** [adriancarriger/angularfire2-offline#36]
I am trying to figure out a way to catch Authentication errors. Getting the following error:
`Uncaught Error permission_denied at Client doesn't have permission to access the desired data`

Using ionic V3.

My workaround for catching this error prior to logging in is to first check for auth.
```
  ionViewWillEnter() {
    this.afAuth.authState.subscribe(user => {
      if (!user) {
        this.reads = Observable.of([]);
        return;
      }
      this.reads = this.afoDb.list('my_fb_list');
    });
  }
```

The lingering issue that is remaining is how properly handle a user logout event.  I'll point out that if I swap out angularfire2-offline for angularfire2 the errors go away, and I can login and logout with any errors.

**Comment by wheredoesyourmindgo** [comment]
Maybe this is fixed by #18

**Comment by wheredoesyourmindgo** [comment]
Definitely related to #18.  Closing ticket. Thanks for including the recommendations in the source code as far as protecting the signOut() when offline & etc.
